src/Bundle/Maker/Factory/ODMDefaultPropertiesGuesser.php

```diff
@@ -40,14 +40,14 @@ public function __invoke(SymfonyStyle $io, MakeFactoryData $makeFactoryData, Mak
                 continue;
             }
 
-            $fieldName = $item['fieldName'];
+            $fieldName = $item['fieldName']; // @phpstan-ignore-line
```

Comment:
this is kinda strange: `$metadata->associationMappings` documented as 
```php
@psalm-var array<string, AssociationFieldMapping>
```
which is a psalm pseudo type refering to an array:
```php
@psalm-type AssociationFieldMapping = array{ ... }
```

but phpstan outputs the following error:
```
Access to offset 'fieldName' on an unknown class Doctrine\ODM\MongoDB\Mapping\AssociationFieldMapping.
```